docroot/modules/custom/va_gov_content_export/src/ListDataCompiler.php

```diff
@@ -67,52 +93,54 @@ public function __construct(EntityTypeManagerInterface $entityTypeManager) {
    */
-  public function updateLists(ContentEntityInterface $entity, TomeExporter $tomeExporter) {
-    $listNids = $this->getListNids($entity);
-    $listNidsToUpdateLater = [];
-    $listEntitiesToUpdateLater = [];
-    if (!empty($listNids)) {
+  public function updateReverseEntityReferenceLists(ContentEntityInterface $entity, TomeExporter $tomeExporter) {
+    $entityReferenceTargetNids = $this->getListNids($entity);
+    $entityReferenceTargetNidsToUpdateLater = [];
+    $targetNodesToUpdateLater = [];
+    if (!empty($entityReferenceTargetNids)) {
       // Process each list this entity should appear in.
-      foreach ($listNids as $listNid => $listFieldName) {
-        $listNidsToUpdateLater[] = $this->updateList($entity, $listNid, $listFieldName);
+      foreach ($entityReferenceTargetNids as $listNid => $listFieldNames) {
+        $entityReferenceTargetNidsToUpdateLater[] = $this->updateList($entity, $listNid, $listFieldNames);
       }
-      $listNidsToUpdateLater = array_filter($listNidsToUpdateLater);
+      $entityReferenceTargetNidsToUpdateLater = array_filter($entityReferenceTargetNidsToUpdateLater);
     }
-    if (!empty($listNidsToUpdateLater)) {
+    if (!empty($entityReferenceTargetNidsToUpdateLater)) {
       // @todo Add a check to bypass this if the bulk export is running.
       // https://github.com/ ... /va.gov-cms/issues/2481.
       // Load the list entities that need to be exported.
-      $listEntitiesToUpdateLater = $this->entityTypeManager->getStorage('node')
-        ->loadMultiple($listNidsToUpdateLater);
+      $targetNodesToUpdateLater = $this->entityTypeManager->getStorage('node')
+        ->loadMultiple($entityReferenceTargetNidsToUpdateLater);
       // Export the lists to have their reverse field list updated.
-      foreach ($listEntitiesToUpdateLater as $listEntityToUpdate) {
+      foreach ($targetNodesToUpdateLater as $targetNodeToUpdateLater) {
         // Update the list export json.
-        $tomeExporter->exportContent($listEntityToUpdate);
+        $tomeExporter->exportContent($targetNodeToUpdateLater);
       }
     }
   }
 
   /**
-   * Update a list to add the list data as ->reverse_field_list.
+   * Update a list to add the list data as ->reverse_fieldname.
    *
    * @param \Drupal\Core\Entity\ContentEntityInterface $entity
    *   The entity is a list or in a list.
    * @param int $listNid
    *   The node id of ths list this entity belongs in.
-   * @param string $listFieldName
-   *   The field machine name to use as the reverse entity reference.
+   * @param array $listFieldNames
+   *   The field machine names to use as the reverse entity reference.
    *
    * @return int
    *   The node id of a list to be updated.
    */
-  public function updateList(ContentEntityInterface $entity, $listNid, $listFieldName) {
+  public function updateList(ContentEntityInterface $entity, $listNid, array $listFieldNames) {
```

Comment:
```suggestion
  public function updateList(ContentEntityInterface $entity, $listNid, array $listFieldNames) : int {
```

type hint the return